src/CameraControls/useCenterGraph.ts

```diff
@@ -44,11 +55,11 @@ export interface CenterGraphOutput {
    * `false` or omitted, the graph will be re-centered regardless of whether the nodes
    * are currently in view.
    */
-  centerNodes: (
-    nodes?: InternalGraphNode[],
-    animate?: boolean,
-    centerOnlyIfNodesNotInView?: boolean
-  ) => void;
+  centerNodes: ({
+    nodes,
```

Comment:
I probably would have made `nodes` the first arg and then all the others a `options` arg. I think this is fine for now though.